src/zenml/integrations/label_studio/annotators/label_studio_annotator.py

```diff
@@ -87,7 +88,10 @@ def get_url(self) -> str:
         Returns:
             The URL of the annotation interface.
         """
-        return f"http://localhost:{self.config.port}"
+        if self.config.instance_url == DEFAULT_LOCAL_INSTANCE_URL:
```

Comment:
Why do we need this condition? Isn't it okay just to return self.config.instance_url?

Reply:
Not quite, but you were right that there was a simpler way to do this. I encountered some errors when testing it out on a deployed instance earlier, but I realised what I was doing wrong. I removed the conditional now and updated the docs to make it clearer what the user needs to do.